group_vars/osds.yml.sample

```diff
@@ -115,22 +116,33 @@ dummy:
 # /dev/sda: PTTYPE="gpt"
 # /dev/sda1: UUID="9c43e346-dd6e-431f-92d8-cbed4ccb25f6" TYPE="xfs" PARTLABEL="ceph data" PARTUUID="749c71c9-ed8f-4930-82a7-a48a3bcdb1c7"
 # /dev/sda2: PARTLABEL="ceph block" PARTUUID="e6ca3e1d-4702-4569-abfa-e285de328e9d"
-#journal_collocation: false
+#
+
+#osd_scenario: dummy
+#valid_osd_scenarios:
+#  - collocated
+#  - non-collocated
```

Comment:
So they are more self-explanatory should we use ``collocated_journals`` and ``dedicated_journals`` here instead?